sdk/python/jobs/pipelines/1b_pipeline_with_python_function_components/src/components.py

```diff
@@ -12,7 +12,8 @@ def train_model(
     max_epochs: int,
     model_output: Output(type="uri_folder"),
     learning_rate=0.02,
-):
+    input_int: int=None
```

Comment:
make up some more meaningful name for the new inputs/ouptuts?

Reply:
Since the inputs and outputs here are not actually used, I only refer to them as int/str to clearly indicate the type of input to the user.

Reply:
Since the inputs and outputs here are not actually used, I only refer to them as int/str to clearly indicate the type of input to the user.

Reply:
done.